cogs/help.py

```diff
@@ -232,7 +232,7 @@ def __init__(self, bot: commands.Bot):
         self.add_item(HelpButton("Python", style=ButtonStyle.green, custom_id="python"))
     
     async def interaction_check(self, interaction: Interaction):
-        if interaction.user.get_role(NO_HELP_ID) is not None:
+        if interaction.user.get_role(HELP_BANNED_ID) is not None:
```

Comment:
....And this is `HELP_BANNED_ID`, which will lead to python saying that this is undefined.